packages/components/src/RecurringSelect/RecurringSelect.tsx

```diff
@@ -0,0 +1,110 @@
+import React from "react";
+import { MonthlySelect } from "./components";
+import { CurrentRecurrenceComponent } from "./CurrentRecurrenceComponent";
+import styles from "./RecurringSelect.css";
+import {
+  DurationPeriod,
+  RecurrenceRule,
+  isMonthly,
+  typeInitialStateMap,
+} from "./types";
+import { Option, Select } from "../Select";
+import { InputNumber } from "../InputNumber";
+import { InputGroup } from "../InputGroup";
+import { Content } from "../Content";
+import { Text } from "../Text";
+
+interface RecurringSelectProps {
+  disabled?: boolean;
+  recurrenceRule: RecurrenceRule;
+  onChange(next: RecurrenceRule): void;
```

Comment:
```suggestion
  onChange(value: RecurrenceRule): void;
```

consistency and also make it easier for people that would be using this to assume what that param be